nbextensions/appCell2/widgets/tabs/resultsTab.js

```diff
@@ -1,193 +1,18 @@
-define([
-    'bluebird',
-    'uuid',
-    'common/ui',
-    'kb_common/html',
-    './status/jobStateList',
-    './resultsViewer',
-], (Promise, Uuid, UI, html, JobStateList, JobResult) => {
+define(['bluebird', './resultsViewer'], (Promise, JobResult) => {
     'use strict';
-    const t = html.tag,
-        div = t('div');
 
     function factory(config) {
         let container,
             model = config.model,
-            ui,
-            jobList,
             resultsViewer;
 
         function start(arg) {
             container = arg.node;
-            model = arg.model;
-            ui = UI.make({ node: container });
-
-            const jobState = model.getItem('exec.jobState');
-            if (jobState.child_jobs && jobState.child_jobs.length) {
-                return startBatch();
-            } else {
-                return startSingle(jobState);
-            }
```

Comment:
`startBatch` would never be called because batch mode does not create jobs with the `child_jobs` attribute populated ==> deleted all unused code